onto_spread_ed/config.py

```diff
@@ -2,6 +2,8 @@
 import os
 
 APP_TITLE = "Ontology Spreadsheet Editor"
+ENVIRONMENT = os.environ.get("FLASK_ENV")
+URL_PREFIX = '/onto-ed'
```

Comment:
I think, the default URL prefix should be an empty string.

Reply:
Hey Björn,
We have made use of this URL_Prefix in some of the code to access it from the config file and if we make it empty then those will break.
For eg: <link rel="stylesheet" href="{{ config['URL_PREFIX'] }}/static/excel-bootstrap-table-filter-style.css">.

Reply:
<link rel="stylesheet" href="{{ config['URL_PREFIX'] }}/static/excel-bootstrap-table-filter-style.css">